docs/source/initialsetup/compilingmame.rst

```diff
@@ -290,47 +304,57 @@ Emscripten Javascript and HTML
+process misses something. E.g.:
 
 The **SOURCES** parameter should have the path to at least one driver **.cpp**
 file.  The make process will attempt to locate and include all dependencies
 necessary to produce a complete build including the specified driver(s).
 However, sometimes it is necessary to manually specify additional files (using
 commas) if this process misses something. e.g.
 
-.. code-block:: bash
-
-    emmake make SUBTARGET=apple2e SOURCES=src/mame/drivers/apple2e.cpp,src/mame/machine/applefdc.cpp
-
-The value of the **SUBTARGET** parameter serves only to differentiate multiple
+The value of the SUBTARGET parameter serves only to differentiate multiple
 builds and need not be set to any specific value.
 
-Emscripten supports compiling to WebAssembly with a JavaScript loader instead of
-all-JavaScript, and in later versions this is actually the default.  To force
-WebAssembly on or off, add **WEBASSEMBLY=1** or **WEBASSEMBLY=0** to the make
-command line.
+Emscripten supports compiling to WebAssembly with a JavaScript loader instead
+of all-JavaScript, and in later versions this is actually the default. To force
+WebAssembly on or off, add WEBASSEMBLY=1 or WEBASSEMBLY=0 to the make command
+line.
 
-Other make parameters can also be used, e.g. **-j** for parallel compilation, as
-described earlier.
+Other make parameters can also be used, e.g. *-j* for multithreaded compilation
+as described earlier.
 
 When the compilation reaches the emcc phase, you may see a number of
-*"unresolved symbol"* warnings.  At the moment, this is expected for
-OpenGL-related functions such as glPointSize.  Any others may indicate that an
-additional dependency file needs to be specified in the **SOURCES** list.
+*"unresolved symbol"* warnings. At the moment, this is expected for
+OpenGL-related functions such as glPointSize. Any others may indicate that an
+additional dependency file needs to be specified in the SOURCES list.
 Unfortunately this process is not automated and you will need to search the
-source tree to locate the files supplying the missing symbols.  You may also be
-able to get away with ignoring the warnings if the code path referencing them is
-not used at run-time.
+source tree to locate the files supplying the missing symbols. You may also be
+able to get away with ignoring the warnings if the code path referencing them
+is not used at run-time.
+
+If all goes well, a .js file will be output to the current directory. This file
+cannot be run by itself, but requires an HTML loader to provide it with a canvas
+to output to and pass in command-line parameters. The
+`Emularity project <https://github.com/db48x/emularity>`_ provides such a
+loader.
+
+There are example .html files in that repository which can be edited to point
+to your newly compiled MAME js filename and pass in whatever parameters you
+desire. You will then need to place all of the following on a web server:
```

Comment:
This seems to have lost some formatting and duplicated some of the following paragraphs.  Could you check it again?

Reply:
Yeah. That's me completely forgetting that command line parameters should always be **bold**.

I need to write a style guide before I do anything more with the docs. Well, after this, anyway.